MediaBrowser.Controller/SystemBackupService/BackupManifestDto.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace MediaBrowser.Controller.SystemBackupService;
+
+/// <summary>
+/// Manifest type for backups internal structure.
+/// </summary>
+public class BackupManifestDto
+{
+    /// <summary>
+    /// Gets or sets the jellyfin version this backup was created with.
+    /// </summary>
+    public required Version JellyfinVersion { get; set; }
```

Comment:
Never use jellyfin name in API names

```suggestion
    public required Version ServerVersion { get; set; }
```